pontos/github/argparser.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 2022 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+""" Argument parser for pontos-github """
+
+from argparse import ArgumentParser, Namespace
+import os
+from typing import List
+
+
+def from_env(name: str) -> str:
+    if name in os.environ:
+        return os.environ[name]
+    else:
+        return name
```

Comment:
environ is a dict like (if not even a real dict) object so this can be simplified
```suggestion
    return os.environ.get(name, name)   
```